core/dpdk.cc

```diff
@@ -108,16 +118,21 @@ void init_eal(int dpdk_mb_per_socket, std::string nonworker_corelist) {
       // Do not bother with /var/run/.rte_config and .rte_hugepage_info,
       // since we don't want to interfere with other DPDK applications.
       "--no-shconf",
+      // TODO(sangjin) switch to dynamic memory mode
+      "--legacy-mem",
```

Comment:
we use this too https://github.com/omec-project/upf-epc/blob/master/patches/bess/0003-legacy-mem.patch
CAP_SYS_NICE otherwise

Reply:
@krsna1729 Based on your own experiment, it looks like `CAP_SYS_ADMIN` is required.  This is not really surprising but doesn't seem to be clear from the man pages.

Reply:
(oops, attached this at the wrong place, that's for the `--iova pa` arguments)